Funcky/Extensions/EnumerableExtensions/EitherPartitions.cs

```diff
@@ -0,0 +1,12 @@
+namespace Funcky.Extensions;
+
+public readonly struct EitherPartitions<TLeft, TRight>
```

Comment:
What about using a readonly record struct, you use Deconstruct

Reply:
I'd prefer to stick with explicit implementations for now, because the public API analyzer doesn't track compiler-generated methods.